src/components/blocks/table/Tbody.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ComponentProps } from 'react';
+import { HtmlComponentProps, ValidReactElement } from 'src/components/html-component-props';
+import Html from 'src/components/blocks/Html';
+import HtmlDataTypes from '../../../../data/types/html';
+
+type Props = {
```

Comment:
As per Table, I think this should be `HtmlComponentProps<'tbody'>` and if data is null or isString, return null.